nengo/spa/bind.py

```diff
@@ -0,0 +1,39 @@
+import nengo
+from nengo.spa.module import Module
+
+
+class Bind(Module):
+    """A module for binding together both inputs
+
+    Parameters
+    ----------
+    dimensions : int
+        Number of dimensions for the two vectors to be compared
+    vocab : Vocabulary, optional
+        The vocabulary to use to interpret the vectors
+    n_neurons : int
+        Number of neurons to use in each product computation
+    invert_a, invert_b : bool
+        Whether to reverse the order of elements in either
+        the first input (`invert_a`) or the second input (`invert_b`).
+        Flipping the second input will make the network perform circular
+        correlation instead of circular convolution.
+    input_magnitude : float
+        Effective input magnitude for the multiplication.
+        The actual input magnitude will be this value times sqrt(2)
```

Comment:
>I'd also be supportive of just removing this parameter altogether, since I can't think of a spa >use case for it.
What's the use case outside of SPA?

As for the general comment, I made the mistake of copying the doc string from product when I should have copied it from CircularConvolution. Will fix.